### PR TITLE
fix(Pagination): fix page size dropdown styles

### PR DESCRIPTION
**Pull request by plagoa**
Applitools errors are expected.

### CODE REVIEW

File: packages/core/src/Table/stories/TableSection/CompleteTableSection.tsx [hunk 1]
```diff
@@ -32,8 +32,9 @@ const EmptyRow = () => (
 );
 
 export const CompleteTableSection = () => {
+  const totalRows = 64;
   const columns = useMemo(() => getColumns(), []);
-  const [data, setData] = useState(makeData(64));
+  const [data, setData] = useState(makeData(totalRows));
```
Comment (zettca) [inline packages/core/src/Table/stories/TableSection/CompleteTableSection.tsx:37]: We should us `data.length`, not `totalRows`. Rows could be added or removed
```suggestion
  const [data, setData] = useState(makeData(64));
```

File: packages/core/src/TableSection/TableSection.styles.tsx [hunk 2]
```diff
@@ -85,6 +85,12 @@ export const { staticClasses, useClasses } = createClasses("HvTableSection", {
     [`& .${paginationClasses.pageSizeOptions}`]: {
       left: theme.space.sm,
     },
+    [`& .${paginationClasses.pageSizeHeader}`]: {
+      border: "none",
```
Comment (zettca) [inline packages/core/src/TableSection/TableSection.styles.tsx:89]: Shouldn't we also remove the border of the popup content? This looks strange...
image

File: packages/core/src/Table/stories/TableSection/CompleteTableSection.tsx [hunk 3]
```diff
@@ -132,7 +133,10 @@ export const CompleteTableSection = () => {
         </HvTable>
       </HvTableContainer>
       {page?.length ? (
-        <HvPagination {...getHvPaginationProps?.()} />
+        <HvPagination
```
Comment (MEsteves22) [inline packages/core/src/Table/stories/TableSection/CompleteTableSection.tsx:136]: I think we should add this new pagination to all samples in the table section page

File: packages/core/src/Table/stories/TableSection/TableEditable.tsx [hunk 4]
```diff
@@ -585,7 +585,13 @@ const Table = <T extends Data>({
         </HvTableContainer>
       </LoadingContainer>
       {page.length > 0 ? (
-        <HvPagination {...getHvPaginationProps?.()} />
+        <HvPagination
+          {...getHvPaginationProps?.()}
+          labels={{
+            pageSizePrev: "",
+            pageSizeEntryName: `of ${(data && data.length * pageCount) || 0}`,
```
Comment (MEsteves22) [inline packages/core/src/Table/stories/TableSection/TableEditable.tsx:592]: Sometimes the page count is wrong (for example if you go to the last page, the page count is 900)

```suggestion
            pageSizeEntryName: `of ${(totalRecords}`,
```

You can get the `totalRecords` from `useServerData`. You'll  need to pass it to the `Table` component as a prop.

**Review by MEsteves22 — APPROVED** [review]
🚀

I just have this last comment before merging since the page count is wrong for the editable table:

File: packages/core/src/Table/stories/TableSection/TableEditable.tsx (hunk 4 above)
Comment (plagoa) [reply packages/core/src/Table/stories/TableSection/TableEditable.tsx]: Got it, forgot about that. Updated.

**Review by HQFOX — APPROVED** [review]

**Pull request merged**